src/checkers/inference/InferenceChecker.java

```diff
@@ -0,0 +1,37 @@
+package checkers.inference;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.List;
+
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.common.basetype.BaseTypeVisitor;
+import org.checkerframework.framework.qual.Unqualified;
+
+import checkers.inference.quals.VarAnnot;
+
+public class InferenceChecker extends BaseTypeChecker {
```

Comment:
I believe this class is here solely to mimic the inheritance structure of the Checker Framework and to provide the INFERENCE_ANNOTATIONS.  It may be smart to fold this into BaseInferrableChecker rather than having it on it's own.  If not, we should add a comment as to its actual purpose.